src/storyblok-auth-api/AuthHandlerParams.ts

```diff
@@ -50,4 +50,8 @@ export type AuthHandlerParams = {
    *  - `https://my-app.my-domain.com/api/authenticate/storyblok/callback` as the OAuth2 callback URL
    */
   endpointPrefix: string | undefined // To make explicit, do not make this optional.
+  /*
+   *  The `customAuthEndpoint` is a custom URL that allows changing the environment where the OAuth flow will run.
+   */
+  customAuthEndpoint?: string
```

Comment:
Can we name it something else?

I mean, everything in the params is "custom"
It's more like the base url right, not the whole endpoint URL?

Maybe `storyblokApiBaseUrl`?






[5:45](https://storyblok.slack.com/archives/C08R0M6RXT2/p1746546333904629)
It's more like the base url right, not the whole endpoint URL?